src/main/java/seedu/taskman/logic/parser/CommandParser.java

```diff
@@ -209,6 +209,25 @@ private Command prepareEdit(String args) {
             return new IncorrectCommand(ive.getMessage());
         }
     }
+    
+    private Command prepareComplete(String args) {
+
+        Optional<Integer> index = parseIndex(args);
+        if(!index.isPresent()){
+            return new IncorrectCommand(
+                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, CompleteCommand.MESSAGE_USAGE));
+        }
+
+        return new CompleteCommand(
+		        index.get(),
+		        null,
```

Comment:
As above, constructor which only takes targetIndex would be more appropriate. Don't need to put the nulls.